spec/unit/puppet/provider/scheduled_task/win32_taskscheduler_spec.rb

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env ruby
 require 'spec_helper'
 
-require 'puppet/util/windows/taskscheduler' if Puppet.features.microsoft_windows?
+require_relative '../../../../legacy_taskscheduler' if Puppet.features.microsoft_windows?
```

Comment:
Should this be in the `spec_helper.rb` and then removed from individual test files? or not worth it?

Reply:
I'm thinking _possibly_ not since it's only needed for two test files, not the whole suite?

Reply:
I prefer it to be in the two files explicitly where it's needed (as it is now), rather than pollute all the other tests.